packages/wonder-blocks-link/src/components/link-core.js

```diff
@@ -43,11 +44,14 @@ export default class LinkCore extends React.Component<Props> {
             ...restProps
         } = this.props;
 
-        const linkStyles = _generateStyles(kind, light, visitable);
+        const linkStyles = _generateStyles(inline, kind, light, visitable);
+        const restingStyles = inline
+            ? linkStyles.restingInline
+            : linkStyles.resting;
 
         const defaultStyles = [
             sharedStyles.shared,
-            !(hovered || focused || pressed) && linkStyles.default,
+            !(hovered || focused || pressed) && restingStyles,
```

Comment:
**question:** I'm not sure if this is an issue that recently surfaced, but I noticed that if we focus on the link, then hover on it and move the mouse out, the focus ring is no longer present (but the link is still focused).

Do you know if a recent change introduced that behavior? or this is something we do on purpose?

For reference:

https://user-images.githubusercontent.com/843075/217328219-12f4c11f-bd08-419d-a262-d852719b088b.mov

Reply:
@jandrade re: focus vs hover, I thought that was the intended behavior based on the previous logic in the code. But I just tested links on Wikipedia to see how this is handled there, and you're absolutely right - the focus ring is still there when hovering and clicking! This should be an easy fix.

I just want to confirm that this sounds okay with @megaroeny.

Reply:
@nishasy @jandrade , so does that mean that we should have variants for focus while hovering/pressing? I don't think the designers would ever use that, but I could try to add specs for that in Figma, at least!

I think it would look poor if it kept the underline, though, right?

Reply:
@megaroeny If you don't want the underline and focus ring at the same time, I can just change the logic so that focus takes priority over hover. 

The examples I looked at so far are:
- Wikipedia: Focus includes both the ring and the underline. Hovering over a focused link doesn't make it look different. Pressing changes the color and keeps the focus ring.
- Google: Focus includes just the focus ring. Hovering over a focused link adds an underline. Pressing removes the focus ring.
- w3: All links are underlined. Focus adds the ring and changes the color. Hovering over a focused link doesn't make it look different. Pressing doesn't make it look different.